src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import Grettings from './components/Greetings';
```

Comment:
Currently, the server is throwing an error, I think you made an error with the `Grettings` spelling that was imported from the greetings component. Kindly check this to make the server run without errors 👍

Reply:
Could you please show me what the error says because everything works just fine on my local machine and I have asked my partners to run the app on their machines and everything works just fine.
Thank you

Reply:
@fortuneonyeka, Am sorry I didn't add the screenshot of the error, since it was a spelling mistake and was clear on repo that was why, I will make sure to edit the changes requested and add a screenshot 🙂

Reply:
@bimbolabuari It's alright, thanks a lot for the feedback. I have made the changes already and I have also submitted for another code review.